packages/babel-types/scripts/generators/docs.mjs

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-const util = require("util");
-const stringifyValidator = require("../utils/stringifyValidator");
-const toFunctionName = require("../utils/toFunctionName");
+import util from "util";
+import stringifyValidator from "../utils/stringifyValidator";
+import toFunctionName from "../utils/toFunctionName";
 
-const types = require("../../");
+import types from "../../";
```

Comment:
I think you'll need to specify the file here, since ESM doesn't have the automatic resolution:
```suggestion
import types from "../../lib/index.js";
```
(also in other files in this folder)